db/migrate/20161118150205_add_user_email_fingerprint.rb

```diff
@@ -23,9 +23,10 @@ def down
   end
 
   def encrypt_user_emails
+    uak = EncryptedEmail.user_access_key
     User.where(encrypted_email: '').each do |user|
       email_address = user.email.present? ? user.email : user.id.to_s
-      ee = EncryptedEmail.new_from_email(email_address)
+      ee = EncryptedEmail.new_from_email(email_address, uak)
```

Comment:
I really really prefer to avoid the word `key` has being so over-used as to be meaningless. Might as well name it `variable`.